### PR TITLE
[Closes #424] Removing coupon updates OrderReviewPage and CheckoutPage

### LINKED ISSUE #424: No visual feedback when removing Voucher on order or order confirmation.
### Step 1: Are you in the right place?

  * For issues or feature requests related to the code **in this repository** file a Github issue.
  * For issues relating to EP-Commerce, or Cloud Ops in relation to the reference store please Open a Support ticket.

### Step 2: Describe your environment

  * Device: Windows Laptop
  * OS version: Windows 10 Pro v1903
  * Storefront pulled version: (Demo Team)
  * Node version: (Demo Team)
  * Docker version: (Demo Team)
  * Docker Compose version: (Demo Team)

### Step 3: Describe the problem:

The button to remove voucher codes on the order and order confirmation page do seem to remove the voucher, but not update the screen. So if someone where to press that button, they so no changes, however their voucher code is removed, the total price potentially goes up because of this. When pressing complete order, this customer could be paying more than shown on their screen.

#### Steps to reproduce:

  1. Add a voucher to a cart
  2. Proceed to the order screen
  3. Remove voucher
  4. Return to cart
  5. Proceed to order confirmation screen
  6. Remove voucher

#### Observed Results:

  3. No refresh or price updates are done
  4. The voucher has at this stage been removed
  6. No refresh or price updates are done

#### Expected Results:

I expect the screen to update the voucher status, price and discounts at step 4 and 6.

#### Relevant Code:

  ```
  // TODO(you): code here to reproduce the problem
  ```

### PR DESCRIPTION
**Pull request by maennis-ep**
Description:

Addresses https://github.com/elasticpath/react-pwa-reference-storefront/issues/424.

Added onChange function to update OrderReviewPage and CheckoutPage when a coupon is removed from the order.

Linting:
- [x] No linting errors

Component Updates:
- [ ] Component package requires update on npm [@elasticpath/store-components](https://www.npmjs.com/package/@elasticpath/store-components)

Tests:
- [ ] E2E tests (npm test run with `e2e`)
- [x] Manual tests

Checkout Page:
1) Add an item to the cart
2) Activate a promotion using a coupon code
3) Click on **Proceed to Checkout**
4) Click the **REMOVE** button next to the coupon code on the right side of the page.
5) The coupon will be removed from the order and the order data will update

Order Review Page:
1) Add an item to the cart
2) Activate a promotion using a coupon code
3) Click on **Proceed to Checkout**
4) Click on **Complete Order**
5) Click the **REMOVE** button next to the coupon code underneath the order line items
6) The coupon will be removed from the order and the order data will update

Documentation:
- [ ] Requires documentation updates

### CODE REVIEW

File: app/src/ep.config.json [hunk 1]
```diff
@@ -1,8 +1,7 @@
 {
   "cortexApi": {
-    "path": "/cortex",
-    "scope": "vestri",
-    "pathForProxy": "http://localhost:9080"
+    "path": "https://vestri-us.epdemos.com/cortex",
```
Comment (shaunmaharaj) [inline app/src/ep.config.json]: Please don't override this configuration

File: app/package.json [hunk 2]
```diff
@@ -79,7 +79,7 @@
     "workbox-webpack-plugin": "^4.3.1"
   },
   "scripts": {
-    "start": "BROWSER=none node scripts/start.js",
+    "start": "node scripts/start.js",
```
Comment (shaunmaharaj) [inline app/package.json]: Was this change intentional?

Comment (maennis-ep) [reply app/package.json]: Nope!  Removing it now. (Windows problems)

File: app/src/ep.config.json (hunk 1 above)
Comment (maennis-ep) [reply app/src/ep.config.json]: I have reverted it back, I added my config file by accident

**Comment by shaunmaharaj** [comment]
Thanks @maennis-ep! merged
